cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/ArmTemplateBuilder.java

```diff
@@ -77,6 +77,28 @@ public String build(String stackName, ArmCredentialView armCredentialView, ArmSt
             model.put("resourceGroupName", armUtils.getCustomResourceGroupName(network));
             model.put("existingVNETName", armUtils.getCustomNetworkId(network));
             model.put("existingSubnetName", armUtils.getCustomSubnetId(network));
+            model.put("userImageName", String.format("https://%s.blob.core.windows.net/%s/%s", rootDiskStorage, ArmStorage.IMAGES, imageName));
+            model.put("osDiskVhdName", String.format("https://%s.blob.core.windows.net/%s/%sosDisk",
+                    rootDiskStorage, armStorage.getDiskContainerName(cloudContext), stackName));
+            model.put("sshKeyPath", String.format("/home/%s/.ssh/authorized_keys", armCredentialView.getLoginUserName()));
+            model.put("ipConfigurationsAddress", String.format("'/frontendIPConfigurations/%sipcn", stackName));
```

Comment:
I guess ' character should occur template validation error near `'${ipConfigurationsAddress}'`